sov-modules/sov-modules-impl/integration-tests/tests/demo/mod.rs

```diff
@@ -40,13 +40,30 @@ trait DispatchCall {
 
 /// Runtime defines modules registered in the rollup.
 // #[derive(Genesis, DispatchCall, DispatchQuery, Client)]
-#[derive(Genesis)]
+// #[derive(Genesis)]
 struct Runtime<C: Context> {
     election: Election<C>,
     value_adder: ValueSetter<C>,
     //..
 }
 
+impl<C: Context> Genesis for Runtime<C> {
```

Comment:
We can remove this and enable #[derive(Genesis)].

Reply:
Done!